src/main/webapp/app/overview/exercise-details/exercise-details-student-actions.component.ts

```diff
@@ -143,8 +143,8 @@ export class ExerciseDetailsStudentActionsComponent {
      * @return {assignedTeamId}
      */
     get assignedTeamId(): number | undefined {
-        const participation = this.exercise.studentParticipations![0];
-        return participation ? participation.team?.id : this.exercise.studentAssignedTeamId;
+        const participation = this.exercise.studentParticipations;
+        return participation !== undefined && participation?.length > 0 ? participation[0].team?.id : this.exercise.studentAssignedTeamId;
```

Comment:
I think you can can simplify this line like this:

```suggestion
        return participation?.length > 0 ? participation[0].team?.id : this.exercise.studentAssignedTeamId;
```

or I think you can do:

```suggestion
        return participation && participation.length > 0 ? participation[0].team?.id : this.exercise.studentAssignedTeamId;
```
with the second suggestion, `participation` will be evaluated to false if it's either `null` or `undefined`

Reply:
you're totally right, thanks for noticing. I overlooked it. 👍